css/amp.css

```diff
@@ -151,7 +151,15 @@ html.i-amphtml-ios-embed {
   overflow: hidden !important;
 }
 
-i-amphtml-sizer {
+.i-amphtml-layout-awaiting-size {
+  overflow: visible !important;
+}
+
+.i-amphtml-layout-awaiting-size > * {
+  display: none !important;
```

Comment:
This is not setting the .i-amphtml-layout-awaiting-size element to display: none. It is setting any children of it to display: none (notice the '> *' part of the selector). This is to ensure that nothing spills outside it while we have overflow: visible.